### DISCUSSION TITLE
Shards Failed message

### DISCUSSION THREAD

**Discussion by cyberconsecurity** [telekom-security/tpotce#1024] [General]
Running the latest version of TPOT and any dashboard I load I am getting 1 of 91 shards failed messages, just keeps scrolling over and over.  Here is the response for the error, also attached and image of the error.  Any help on issue would be great or it might just take a reinstall to resolve.

![shards_failed](https://user-images.githubusercontent.com/8916158/155376548-dfcf5259-ac22-4eb4-a0bb-6a7637d3933b.PNG)

![shards_failed_1PNG](https://user-images.githubusercontent.com/8916158/155376715-0017135a-b870-47ee-8566-bb8d3eaffa81.PNG)

{
  "took": 221,
  "timed_out": false,
  "_shards": {
    "total": 91,
    "successful": 90,
    "skipped": 88,
    "failed": 1,
    "failures": [
      {
        "shard": 0,
        "index": "logstash-2021.12.30",
        "node": null,
        "reason": {
          "type": "no_shard_available_action_exception",
          "reason": null,
          "index_uuid": "Y19ZsAeCT6SjIgkVuIl7Sw",
          "shard": "0",
          "index": "logstash-2021.12.30"
        }
      }
    ]
  },
  "hits": {
    "total": 191317,
    "max_score": null,
    "hits": []
  }
}

**Comment by t3chn0m4g3** [comment]
Just delete the index in question, as it seems not accessible.

**Reply by cyberconsecurity** [reply]
Thanks @t3chn0m4g3 that did it, simple fix.
